src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -692,8 +692,9 @@ enum SMART_ACTION
     SMART_ACTION_CU_ENCOUNTER_START                 = 222,    // Resets cooldowns on all targets and removes Heroism debuff(s)
     SMART_ACTION_DO_ACTION                          = 223,    // ActionId
     SMART_ACTION_ATTACK_STOP                        = 224,    //
+    SMART_ACTION_SET_GUID                           = 225,    // Sends the invoker's ObjectGuid to target
```

Comment:
> Sends the invoker's ObjectGuid to target

The invoker would mean the object that triggered the event (player selecting a gossip option, for example)
Source is the object that owns the script